atlas/src/shared/components/VideoPlayer/videoJsPlayer.ts

```diff
@@ -76,5 +78,21 @@ export const useVideoJsPlayer: VideoJsPlayerHook = ({ fill, fluid, height, src,
     player.fill(Boolean(fill))
   }, [player, fill])
 
+  useEffect(() => {
+    if (!player) {
+      return
+    }
+
+    player.muted(muted)
+  }, [player, muted])
+
+  useEffect(() => {
+    if (!player || !posterUrl) {
+      return
+    }
+
+    player.poster(posterUrl)
```

Comment:
Since this is setting a poster, I think a `useLayoutEffect` would be more effective here. wdyt?

Reply:
I don't think it's needed. This would ensure that we can read DOM and synchronously re-render, but it works fine as it is

Reply:
Let's keep this as is then